Core/HLE/sceUsbGps.cpp

```diff
@@ -15,24 +15,57 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
+#include "base/NativeApp.h"
 #include "Core/HLE/HLE.h"
 #include "Core/HLE/FunctionWrappers.h"
 #include "Core/HLE/sceUsbGps.h"
+#include "Core/GPS.h"
 
-enum UsbStatus {
+enum GpsStatus {
 	GPS_STATE_OFF = 0,
 	GPS_STATE_ACTIVATING1 = 1,
 	GPS_STATE_ACTIVATING2 = 2,
 	GPS_STATE_ON = 3,
 };
 
+GpsStatus gpsStatus = GPS_STATE_OFF;
```

Comment:
This is state, so if you want save states to work correctly this needs to be saved. Implement a DoState function like most other HLE modules do.

Reply:
done